app/components/answer-step.jsx

```diff
@@ -0,0 +1,117 @@
+// https://github.com/EnCiv/civil-pursuit/issues/102
+import React, { useState, useEffect } from 'react'
+import { createUseStyles } from 'react-jss'
+import cx from 'classnames'
+import StepIntro from './step-intro'
+import WhyInput from './why-input'
+import ObjectId from 'bson-objectid'
+
+const AnswerStep = props => {
+  const {
+    className = '',
+    intro = '', // text for the introduction
+    whyQuestion1 = '', // the text for the left of the first why input
+    whyQuestion2 = '', // the text for the left of the second why input
+    question = {}, // a pointObj
+    shared = {
+      startingPoint: {},
+      whyMosts: [],
+    },
+    onDone = () => {},
+    ...otherProps
+  } = props
+
+  const classes = useStylesFromThemeFunction(props)
+  const { startingPoint: initialStartingPoint, whyMosts } = shared
+  const [startingPoint, setStartingPoint] = useState(initialStartingPoint)
+  const whyPoints =
+    Object.keys(initialStartingPoint).length === 0
+      ? whyMosts
+      : whyMosts.filter(point => point.parentId === initialStartingPoint._id)
+  const [whyMostInputs, setWhyMostInputs] = useState(whyPoints)
```

Comment:
although whyMosts is an array, you can expect that there will be 0 or 1 entries with a parentId that matches the startingPoint.

I suppose you could console.log an error if more than one is found, and just user the first one.